crates/torii/core/src/sql.rs

```diff
@@ -172,7 +170,11 @@ impl Sql {
         let namespaced_name = entity.name();
         let (model_namespace, model_name) = namespaced_name.split_once('-').unwrap();
 
-        let entity_id = format!("{:#x}", poseidon_hash_many(&keys));
+        let entity_id = format!(
+            "{}{:#x}",
```

Comment:
Why should we prefix `event` here? Do we have conflicts possibilities?
As we want them to behave like entities, why not using the exact same way to compute the `entity_id`?